examples/example.md

```diff
@@ -1,136 +1,160 @@
-# Crossplane Provider IONOS Cloud - Example (PoC)
+# Crossplane Provider IONOS Cloud Usage Example
 
 ## Overview
 
 Crossplane allows the user to manage infrastructure directly from Kubernetes. Crossplane extends a Kubernetes cluster to
 support orchestrating any infrastructure or managed service. Providers extend Crossplane to enable infrastructure
-resource provisioning of specific API.
+resource provisioning of specific APIs.
 
 Crossplane Provider IONOS Cloud contains a Controller and Custom Resource Definitions(CRDs). The CRDs are defined in
-sync with the API and contain the desired state, the Controller has a reconcile loop, and it constantly compares the
-desired state vs actual state and takes action to reach the desired state. Using the SDK Go, the controller performs
-CRUD operations and resources are managed in IONOS Cloud. See [diagram](diagram.png).
+sync with the API and contain the desired state. The Controller has a reconcile loop, and it constantly compares the
+desired state vs the actual state and takes action to reach the desired state. Using the SDK Go, the Controller performs
+CRUD operations and resources are managed in the IONOS Cloud.
 
-In this Proof of Concept of the IONOS Cloud Provider, we will create a DBaaS Postgres Cluster resource in IONOS Cloud.
+In this Proof of Concept of the IONOS Cloud Provider, we will create a DBaaS Postgres cluster resource in the IONOS Cloud.
 
 ## Prerequisites
 
-List of prerequisites:
+Ensure that you have the following:
 
-* A Kubernetes implementation like [`kind`](https://kind.sigs.k8s.io/)
-* [Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
-* [Helm](https://helm.sh/docs/intro/install/)
+* A Kubernetes implementation, such as [`kind`](https://kind.sigs.k8s.io/)
+* [``Kubectl``](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [``Helm``](https://helm.sh/docs/intro/install/)
 * Docker
 * Credentials to access IONOS Cloud
 * Clone this repository locally to be able to run examples
 
-Check prerequisites:
+### Check prerequisites
 
-- K8s (in case of using kind):
+You can now check your prerequisites. To check K8s, in case of using ``kind``, run the following command:
 
 ```bash
 kind version
 ```
 
-- credentials:
+To check the **credentials**, run the following command:
 
 ```bash
 export IONOS_USERNAME=xxx
 export IONOS_PASSWORD=xxx
 export BASE64_PW=$(echo -n "${IONOS_PASSWORD}" | base64)
 ```
 
-or
+OR
 
 ```bash
 export IONOS_TOKEN=xxx
 ```
 
-- clone this repository locally:
+To clone the repository locally, run the following command:
 
 ```bash
 git clone https://github.com/ionos-cloud/crossplane-provider-ionoscloud.git
 cd crossplane-provider-ionoscloud
 ```
 
-## Setup Crossplane Provider IONOS Cloud
+## Set up Crossplane Provider IONOS Cloud
 
-**1. Create a K8s cluster (in case of using kind):**
```

Comment:
Because this a procedure, I would suggest retaining the numbers.

Reply:
Sure, as discussed adding it instead of  using bullets.